Formula/fontforge.rb

```diff
@@ -16,71 +16,72 @@ class Fontforge < Formula
 
   deprecated_option "with-gif" => "with-giflib"
 
-  # Autotools are required to build from source in all releases.
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
+  if build.head?
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+
+    resource "gnulib" do
+      url "git://git.savannah.gnu.org/gnulib.git",
+        :revision => "29ea6d6fe2a699a32edbe29f44fe72e0c253fcee"
+    end
+  end
+
+  depends_on :x11 => :optional
+  if build.with? "x11"
+    depends_on "cairo" => "with-x11"
+    depends_on "pango" => "with-x11"
+  else
+    depends_on "cairo"
+    depends_on "pango"
+  end
   depends_on "pkg-config" => :build
   depends_on "libtool" => :run
   depends_on "gettext"
-  depends_on "pango"
   depends_on "zeromq"
   depends_on "czmq"
-  depends_on "cairo"
   depends_on "fontconfig"
   depends_on "libpng" => :recommended
   depends_on "jpeg" => :recommended
   depends_on "libtiff" => :recommended
+  depends_on "libspiro" => :recommended
+  depends_on "libuninameslist" => :recommended
```

Comment:
I'm hesitant to do this. They *are* distinct libraries and if I understand `resouce` correctly, it's just odd that you'd shift the creation of a *separate* entity/library into the formula for a dependant library - this goes against practically every package management system out there. 

IIRC, GEGL can use libspiro, although the current formula doesn't reference this.